language/de.json

```diff
@@ -0,0 +1,372 @@
+{
+	"Preview": "Vorschau",
+	"Gallery": "Galerie",
+	"Generate": "Generieren",
+	"Skip": "Überspringen",
+	"Stop": "Stop",
+	"Input Image": "Eingabebild",
+	"Advanced": "Erweitert",
+	"Upscale or Variation": "Vergrößern oder Abwandeln",
+	"Image Prompt": "Bild Prompt",
+	"Inpaint or Outpaint (beta)": "Inpainting oder Outpainting (beta)",
+	"Drag above image to here": "Ziehen Sie das obere Bild hierher",
+	"Upscale or Variation:": "Vergrößern oder Abwandeln:",
+	"Disabled": "Deaktiviert",
+	"Vary (Subtle)": "Variieren (Subtil)",
+	"Vary (Strong)": "Variieren (Stark)",
+	"Upscale (1.5x)": "Vergrößern (1.5x)",
+	"Upscale (2x)": "Vergrößern (2x)",
+	"Upscale (Fast 2x)": "Vergrößern (Schnell 2x)",
+	"\ud83d\udcd4 Document": "\uD83D\uDCD4 Dokument",
+	"Image": "Bild",
+	"Stop At": "Anhalten bei",
+	"Weight": "Gewicht",
+	"Type": "Typ",
+	"PyraCanny": "PyraCanny",
+	"CPDS": "CPDS",
+	"* \"Image Prompt\" is powered by Fooocus Image Mixture Engine (v1.0.1).": "* \"Bild Prompt\" wird von Fooocus Bildmischmotor (v1.0.1) angetrieben.",
+	"The scaler multiplied to positive ADM (use 1.0 to disable).": "Skalierung multipliziert mit positivem ADM (1.0 verwenden, um zu deaktivieren).",
+	"The scaler multiplied to negative ADM (use 1.0 to disable).": "Skalierung multipliziert mit negativem ADM (1.0 verwenden, um zu deaktivieren).",
+	"When to end the guidance from positive/negative ADM.": "Wann die Anleitung von positivem/negativem ADM beendet werden soll.",
+	"Similar to the Control Mode in A1111 (use 0.0 to disable).": "Ähnlich wie der Steuerungsmodus in A1111 (0.0 verwenden, um zu deaktivieren).",
+	"Outpaint Expansion (": "Outpaint-Erweiterung (",
+	"Outpaint": "Outpaint",
+	"Left": "Links",
+	"Right": "Rechts",
+	"Top": "Oben",
+	"Bottom": "Unten",
+	"* \"Inpaint or Outpaint\" is powered by the sampler \"DPMPP Fooocus Seamless 2M SDE Karras Inpaint Sampler\" (beta)": "* \"Aus- oder Übermalen\" wird vom Sampler \"DPMPP Fooocus Nahtloser 2M SDE Karras Inpaint Sampler\" (Beta) angetrieben",
+	"Setting": "Einstellung",
+	"Style": "Stil",
+	"Performance": "Leistung",
+	"Speed": "Geschwindigkeit",
+	"Quality": "Qualität",
+	"Aspect Ratios": "Seitenverhältnisse",
+	"width × height": "Breite × Höhe",
+	"Image Number": "Bildnummer",
+	"Negative Prompt": "Negativer Prompt",
+	"Describing what you do not want to see.": "Beschreibe, was du nicht sehen möchtest.",
+	"Random": "Zufällig",
+	"Seed": "Seed",
```

Comment:
Suggestion for "seed": Zufalls Startwert.

Reply:
This suggestion has now been implemented, but I have kept the Seed in brackets as it is a technical term